lib/clouds/libcloud_common.py

```diff
@@ -593,11 +692,11 @@ def get_instances(self, obj_attr_list) :
         '''
         try :
             _status = 100
-            _msg = "cloud_vm_name " + obj_attr_list["cloud_vm_name"]
-            _msg += " from " + self.get_description() + " \"" + obj_attr_list["cloud_name"]
+            _msg = "cloud_vm_name " + obj_attr_list["log_string"]
```

Comment:
Can the behavior of these "log_strings" be done inside the cbdebug print function as a keyword argument instead of being required to be embedded in the message?

Reply:
This has been bothering me for a while too! Yes, I really like the idea of embedding those on cbdebug (it is yet another example of how the continuous use makes more clear the need for modifications). However, I would like to address it on a separate PR. Fair?

Reply:
sounds good.